pkg/controller/controller_utils.go

```diff
@@ -601,7 +601,7 @@ func (r RealPodControl) DeletePod(namespace string, podID string, object runtime
 	if err != nil {
 		return fmt.Errorf("object does not have ObjectMeta, %v", err)
 	}
-	klog.V(2).Infof("Controller %v deleting pod %v/%v", accessor.GetName(), namespace, podID)
+	klog.V(2).InfoS("Deleting pod", "controller", accessor.GetName(), "pod", klog.KRef(namespace, podID))
```

Comment:
This re-factoring, although seems to make the code more readable actually is not. Since it requires the arguments to always be even, to properly create kev - value pairs. The change to `klog.KRef` is fine, but I'd prefer to stay away from using `InfoS`.

Reply:
Hey Maciek,
This interface was selected and approved by SIG instrumentation and SIG architecture in this KEP https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
`InfoS` is meant to replace `Infof` within k8s codebase. If you don't agree that we should use this keyValue interface then let's discuss it in some issue. Would be useful if you proposed alternative structured interface. 

We should not block structured logging migration to discuss interface that was already approved.